ERCS/erc-6538.md

```diff
@@ -2,9 +2,9 @@
 eip: 6538
 title: Stealth Meta-Address Registry
 description: A registry to map addresses to stealth meta-addresses
```

Comment:
Your description doesn't introduce new information that isn't already in the title.

Reply:
Updated the description in https://github.com/ethereum/ERCs/pull/259/commits/f7a4aec74220455e12006d99ce1fae7b94655bbc